### PR TITLE
Added Logout Button in Nav

### PR DESCRIPTION
**Pull request by Ayushi-0903**
Added Logout button in nav:

Screen Shot 2019-12-10 at 7 56 31 PM

Screen Shot 2019-12-10 at 4 33 12 PM

### CODE REVIEW

**Comment by ab7289** [comment]
It looks like this PR is merging changes from the logout_button branch to the search_styling2 branch. Is that intentional?

File: mainapp/templates/nav.html [hunk 1]
```diff
@@ -62,11 +62,18 @@
...
                         {% if request.user.is_authenticated %}
-                        <a class="nav-link" href="/account">My Account ({{ request.user.username}})</a>
+                            <li class="nav-item dropdown " >
+                                <a class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Account ({{ request.user.username}})</a>
+                                <div class="dropdown-menu">
+                                    <a class="dropdown-item p-2 mb-1 bg-dark text-white" href="/account">My Account</a>
+                                    <div class="dropdown-divider"></div>
+                                    <a class="dropdown-item p-2 mb-1 bg-dark text-white" href="{% url 'account_logout' %}">Logout</a>
+                                 </div>
+                            </li>
```
Comment (ab7289) [inline mainapp/templates/nav.html]: So I took a look at the bootstrap docs and it doesn't look like they include an easy way to customize the dropdown menu, so unfortunately you're probably going to have to write some css.

I did find this link though with some examples of customized bootstrap dropdowns that may be helpful in giving you an idea of what CSS modifications you will need. https://www.jquery-az.com/7-customized-bootstrap-dropdown-demos-in-menus-navbar-and-tabs/

Comment (Ayushi-0903) [reply mainapp/templates/nav.html]: Done

**Comment by ab7289** [comment]
@Ayushi-0903  looks good, but i it looks like it no longer displays the currently logged-in user name on the account icon. I don't know if we should be getting rid of that since it was useful in allowing the user to quickly check which account they are logged in as.

Also it looks like this PR is merging into `styling_search2` and not `develop`. Just FYI.
Screen Shot 2019-12-10 at 5 17 44 PM

**Comment by Ayushi-0903** [comment]
Hey Alex, fixed the user name. Can we can merge this into styling_search2 and then finally merge into master ?

File: mainapp/templates/nav.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {% load static %}
 
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" xmlns:background-color="http://www.w3.org/1999/xhtml">
```
Comment (ab7289) [inline mainapp/templates/nav.html]: What does this addition do?

**Pull request merged**